Voycar.Api.Web.Tests.Integration/App.cs

```diff
@@ -5,9 +5,16 @@
 using Microsoft.Extensions.DependencyInjection;
 using Context;
 using Setup;
-using DotNet.Testcontainers.Builders;
 using Testcontainers.PostgreSql;
 
+/* Normally the SUT (API and database) will only be created once for all test-classes and cached afterward.
+ Disabling the WAF cache will create a new instance for every test-class resulting in worse test performance.
+ This behaviour is intended so that test-classes can perform operations on the database without impacting
+ any other tests which assert the content of the database.
```

Comment:
```suggestion
/*
Usually, the SUT (API and database) will only be created once for all test classes and cached afterward.
 Disabling the WAF cache will create a new instance for every test class, resulting in worse test performance.
 This behavior is intended so that test classes can perform operations on the database without impacting
 any other tests that assert the content of the database.
```
Hauptsächlich Vorschläge von Language Tool